docs/source/projections/tmerc.rst

```diff
@@ -1,10 +1,195 @@
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| Transverse Mercator                 | World wide                                          | Various                        | less than 6°                             | Various      |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| Transverse Mercator south oriented  | Southern Africa                                     | 2° intervals E of 11°E         | 2°                                       | 1.000        |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| UTM North hemisphere                | World wide equator to 84°N                          | 6° intervals E & W of 3° E & W | Always 6°                                | 0.9996       |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| UTM South hemisphere                | World wide north of 80°S to equator                 | 6° intervals E & W of 3° E & W | Always 6°                                | 0.9996       |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| Gauss-Kruger                        | Former USSR, Yugoslavia, Germany, S. America, China | Various, according to area     | Usually less than 6°, often less than 4° | 1.0000       |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+| Gauss Boaga                         | Italy                                               | Various, according to area     | 6°                                       | 0.9996       |
++-------------------------------------+-----------------------------------------------------+--------------------------------+------------------------------------------+--------------+
+
+
+
+Example using Gauss-Kruger on Germany area (aka EPSG:31467) ::
+
+    $ echo 9 51 | proj +proj=tmerc +lat_0=0 +lon_0=9 +k=1 +x_0=3500000 +y_0=0 +ellps=bessel +datum=potsdam +units=m +no_defs
+    3500000.00	5651505.56
+
+Example using Gauss Boaga on Italy area (EPSG:3004) ::
+
+    $ echo 15 42 | proj +proj=tmerc +lat_0=0 +lon_0=15 +k=0.9996 +x_0=2520000 +y_0=0 +ellps=intl +units=m +no_defs
+    2520000.00	4649858.60 
+
+Mathematical definition
+#######################
+
+The formulas describing the Transverse Mercator are all taken from proj4 sources.
```

Comment:
I believe the formulas used in tmerc are based on Snyders work. So credit were credit is due, I guess...

It might actually be a good idea to have a look at those and see if they describe the maths more clearly. Most times what's in the code has been rewritten because of numerical concerns.